launch-game.sh

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
-# Note: this relies on the non-standard -f flag implemented by gnu readlink
-mono OpenRA.Game.exe Engine.LaunchPath="$(readlink -f $0)" "$@"
-if [ $? != 0 -a $? != 1 ]
+MODLAUNCHER=$(python -c "import os; print(os.path.realpath('$0'))")
```

Comment:
gnu readlink is *already* an extra dependency, and doesn't work on OSX or other systems that use BSD readlink.